create_tables.sql

```diff
@@ -1,112 +1,111 @@
-
```

Comment:
doesn't look like you changed anything here - you should check your git settings to make sure you're not committing CRLF line endings so we don't get these massive diffs